java/app-encryption/src/it/java/com/godaddy/asherah/regression/DynamoDbGlobalTableIT.java

```diff
@@ -71,24 +71,46 @@ void testRegionSuffixBackwardCompatibility() {
     byte[] decryptedBytes;
```

Comment:
We should probably add another test where we encrypt with suffix and decrypt without suffix.